readme.md

```diff
@@ -213,7 +213,7 @@ To create a batch, write a batch definition (see the
 [example batch definition][example-batch-definition]), create some data
 in the [JSON Lines][json-lines] format, and then create the batch using
 `amti create-batch`. Use the `-h` option for details. You can find some
-example data in the [`data.jsonl`][data-jsonl] file.
```

Comment:
This link is actually reference-style (see [here](https://daringfireball.net/projects/markdown/syntax#link) in the markdown spec). So, the place to fix the broken link would be line 227: `[data-jsonl]: ./examples/test/data.jsonl`.